packages/theme/src/BackstageTheme.ts

```diff
@@ -66,6 +69,7 @@ const extendedThemeConfig = {
         default: COLORS.DEFAULT_PAGE_THEME_COLOR,
       },
     },
+    // @ts-ignore
```

Comment:
the bad news in this PR: making the types work in this file was proving annoying and I decided to nope out on the last few cases where it seems like some annotations are needed to convince the type system that this is valid.